docs/overview.md

```diff
@@ -0,0 +1,100 @@
+# Cumulus overview
+
+This document provides high-level documentation for Cumulus.
+
+## Runtime
+
+Each Substrate blockchain provides a runtime. The runtime is the state transition function of the
+blockchain. Cumulus provides interfaces and extensions to convert a default Substrate runtime into a
```

Comment:
Is "default" the word we want to use here?

Do we mean any runtime built with FRAME, or are the requirements more extreme?

Reply:
Changed it to `Substrate FRAME runtime`